app/controllers/stories_controller.rb

```diff
@@ -0,0 +1,40 @@
+class StoriesController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  def index
+    @stories = Story.all
+    render json: @stories
+  end
+
+  def show
+    @story = Story.find(params[:id])
+    render json: @story
+  end
+
+  def create
+    @story = Story.new(story_params)
```

Comment:
you probably want more protection around this but depends on how secure you want to make this project